spec/features/event/new_event_spec.rb

```diff
@@ -11,42 +11,35 @@
     @user.destroy
   end
 
-  it "should render without an error" do
+  it 'should render without an error' do
     visit new_event_path
   end
 
-  it "should be possible to enter date conditions for an event" do
+  it 'should be possible to enter date conditions for an event' do
     visit new_event_path
 
     expect(page).to have_field('event_deadline')
     expect(page).to have_field('event_startdate')
     expect(page).to have_field('event_enddate')
   end
 
-  it "should have a field for event duration" do
+  it 'should have a field for event duration' do
     visit new_event_path
 
     expect(page).to have_field('event_duration')
   end
 
-  it "should have a dropdown menu for type" do
-    visit new_event_path
-
-    expect(page).to have_select('event_type', options: [I18n.t('events.new.select_type'), 'Tournament', 'League'])
-  end
-
-  it "should be possible to create a date conditions for an event" do
-    visit new_event_path
-    fill_in "event_name", with: "name"
-    fill_in "event_discipline", with: "soccer"
-    select "Tournament", from: "event_type"
-    select "Ko", from: "event_game_mode_tournament"
-    select "Single", from: "event_player_type"
-    fill_in "event_max_teams", with: "5"
-
-    fill_in "event_deadline", with: Date.tomorrow.to_s
-    fill_in "event_startdate", with: (Date.tomorrow + 2).to_s
-    fill_in "event_enddate", with: (Date.tomorrow + 3).to_s
+  it 'should be possible to create a date conditions for an event' do
+    visit new_tournament_path
+    fill_in Event.human_attribute_name(:name), with: 'name'
```

Comment:
is this really good? isn't here `I18n` missing?

Reply:
`human_attribute_name` already translates the name, according to its behaviour/docs.